### PR TITLE
 APEX 329 - Continue refactor TestApexUser 

### PR DESCRIPTION
**Pull request by babinskizoltan**
Fixed some CRs. Premine wallets used in tests.

# Description

Please provide a detailed description of what was done in this PR

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [ ] I have assigned this PR to myself
- [ ] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [ ] I have tested this code manually

### Manual tests

Please complete this section if you ran manual tests for this functionality, otherwise delete it

# Documentation update

Please link the documentation update PR in this section if it's present, otherwise delete it

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

**Comment by babinskizoltan** [comment]
NOT DONE: e2e-polybft/e2e/nexus_bridge_test.go

File: .github/workflows/apex-ci.yml [hunk 1]
```diff
@@ -128,7 +128,7 @@ jobs:
       github.event_name == 'pull_request'
     with:
       e2e_apex_big_test: ${{ inputs.e2e_apex_big_test || 'false' }}
-      e2e_apex_skip_redundant_tests: ${{ inputs.e2e_apex_skip_redundant_tests || 'true' }}
+      e2e_apex_skip_redundant_tests: ${{ inputs.e2e_apex_skip_redundant_tests || 'false' }}
```
Comment (babinskizoltan) [inline .github/workflows/apex-ci.yml]: don't forget to revert this

File: e2e-polybft/e2e/nexus_bridge_test.go [hunk 2]
```diff
@@ -2348,19 +2008,18 @@ func TestE2E_ApexBridgeWithNexus_BatchFailed(t *testing.T) {
...
 
-func convertToEthValues(sendAmount uint64) (uint64, *big.Int) {
-	sendAmountDfm := new(big.Int).SetUint64(sendAmount)
-	exp := new(big.Int).Exp(big.NewInt(10), big.NewInt(6), nil)
-	sendAmountDfm.Mul(sendAmountDfm, exp)
+		err = apex.WaitForExactAmount(ctx, user, cardanofw.ChainIDNexus, ethExpectedBalance, 100, time.Second*10)
 
-	return sendAmountDfm.Uint64(), ethgo.Ether(sendAmount)
+		// temp - will remove this
+		if err != nil {
```
Comment (babinskizoltan) [inline e2e-polybft/e2e/nexus_bridge_test.go]: delete this if later

**Comment by babinskizoltan** [comment]
big tests haven't been tested yet

**Comment by babinskizoltan** [comment]
Tried manually running CI with only Build Blade, Lint and Big Tests selected, however, big tests were not run for some reason.

CI run: [here](https://github.com/Ethernal-Tech/blade-apex-bridge/actions/runs/11201720971)

File: e2e-polybft/cardanofw/test-cardano-cluster.go [hunk 3]
```diff
@@ -233,6 +279,35 @@ func (c *TestCardanoCluster) NewTestServer(id int, port int) error {
 	return err
 }
 
+func GetLogsFile(t *testing.T, filePath string, withStdout bool) io.Writer {
```
Comment (igorcrevar) [inline e2e-polybft/cardanofw/test-cardano-cluster.go]: put this in in utils

File: e2e-polybft/e2e/nexus_bridge_test.go [hunk 4]
```diff
@@ -12,13 +11,17 @@ import (
...
 	"time"
 
-	ethwallet "github.com/0xPolygon/polygon-edge/consensus/polybft/wallet"
 	"github.com/0xPolygon/polygon-edge/e2e-polybft/cardanofw"
 	"github.com/Ethernal-Tech/cardano-infrastructure/wallet"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/umbracle/ethgo"
-)
+	/*
```
Comment (igorcrevar) [inline e2e-polybft/e2e/nexus_bridge_test.go]: remove all commented out imports

File: e2e-polybft/e2e/nexus_bridge_test.go [hunk 5]
```diff
@@ -395,10 +317,12 @@ func TestE2E_ApexBridgeWithNexus_NtP_ValidScenarios(t *testing.T) {
 			go func(receiverIdx int) {
 				defer wgResults.Done()
 
-				err := cardanofw.WaitForAmount(context.Background(), txProviderPrime, cardanoAddresses[receiverIdx], func(val uint64) bool {
-					return val == expectedAmountOnPrime
-				}, 100, time.Second*10)
-				require.NoError(t, err)
+				expectedAmount := new(big.Int).SetUint64(sendAmountDfm)
```
Comment (igorcrevar) [inline e2e-polybft/e2e/nexus_bridge_test.go]: this could be some utils function

**Review by igorcrevar — APPROVED** [review]

**Pull request merged**